datalad/plugin/tests/test_addurls.py

```diff
@@ -430,8 +431,17 @@ def get_annex_commit_counts():
 
         n_annex_commits = get_annex_commit_counts()
 
-        ds.addurls(self.json_file, "{url}", "{name}")
 
+        # Meanwhile also test that we can specify path relative
+        # to the top of the dataset, as we generally treat paths in
+        # Python API, and it will be the one saved in commit
+        # message record
+        json_file = op.relpath(self.json_file, ds.path)
+        ds.addurls(json_file, "{url}", "{name}")
+        # On Windows, the most recent commit will be the 'adjusted branch'
+        # commit of git-annex, therefore we need to take a look into the
+        # default branch
+        ok_startswith(ds.repo.format_commit('%b', DEFAULT_BRANCH), f"url_file='{json_file}'")
```

Comment:
Sorry, the pain of two branches.  This part is only relevant for master, after 60547c229a (ENH: addurl - store original urlfile, not resolved full path, 2020-10-24).

Reply:
ahh, I see. thanks. I'll remove this change then. :)